docs/core/diagnostics/event-counter-perf.md

```diff
@@ -0,0 +1,100 @@
+
+When dealing with a large number of events, knowing the measure per event is not useful either. Most of the time all you need is just some statistics out of it. So you could crank the statistics within the process itself and then write an event once in a while to report the statistics, that's what <xref:System.Diagnostics.Tracing.EventCounter> will do. Let's take a look at an example how to do this in <xref:System.Diagnostics.Tracing.EventSource?displayProperty=fullName>.
+
+```csharp
+// Give your event sources a descriptive name using the EventSourceAttribute, otherwise the name of the class is used.
+[EventSource(Name = "Samples-EventCounterDemos-Minimal")]
+public sealed class MinimalEventCounterSource : EventSource
+{
+    // Define the singleton instance of the event source
+    public static MinimalEventCounterSource Log = new MinimalEventCounterSource();
+    private EventCounter RequestCounter;
+
+    private MinimalEventCounterSource() : base(EventSourceSettings.EtwSelfDescribingEventFormat)
+    {
+        RequestCounter = new EventCounter("request", this);
+    }
+
+    // Indicate that a request for a URL was made which took a particular amount of time
+    public void Request(string url, float elapsedMSec)
+    {
+        // Notes:
+        //   1. the event ID passed to WriteEvent (1) corresponds to the (implied) event ID
+        //      assigned to this method. The event ID could have been explicitly declared
+        //      using an EventAttribute for this method
+        //   2. Each counter supports a single float value, so conceptually it maps to a single
+        //      measurement in the code.
+        //   3. You don't have to have log with WriteEvent if you don't think you will ever care about details
+        //       of individual requests (that counter data is sufficient).
+
+        // This logs it to the event stream if events are on.
+        WriteEvent(1, url, elapsedMSec);
+        // This adds it to the PerfCounter called 'Request' if PerfCounters are on
+        RequestCounter.WriteMetric(elapsedMSec);
+    }
+}
+```
+
+The <xref:System.Diagnostics.Tracing.EventSource.WriteEvent%2A?displayProperty=nameWithType> line is the <xref:System.Diagnostics.Tracing.EventSource> part and is not part of <xref:System.Diagnostics.Tracing.EventCounter>, it was written to show that you can log a message together with the event counter.
+
+You logged the metric to the <xref:System.Diagnostics.Tracing.EventCounter>, but unless you can actually get the statistics out of it, it is not useful. To get the statistics, you need to enable the <xref:System.Diagnostics.Tracing.EventCounter> by creating a timer that fires as frequently as you want the events, as well as a listener to capture the events. To do that, you can use PerfView.
+
+There is an extra keyword that you will need to turn on the EventCounters.
```

Comment:
I'd recommend doing the demo in dotnet-counters so that it will work cross-platform. A major part of the value proposition for EventCounters is that it would be available xplat, so demoing it in a Windows-only tool may create confusion. I also wouldn't expect PerfView to be a primary tool for analyzing counter data in the long-run, it just happens to be the only tool that existed at the time this sample was first written.